include/glow/Runtime/RuntimeTypes.h

```diff
@@ -143,6 +143,18 @@ struct DeviceConfig {
       : backendKind(kind), name(name), parameters(parameters) {}
 
   bool hasName() const { return name != ""; }
+
+  void setDeviceSize(uint64_t mem_size) {
```

Comment:
Let's call these `{set,get}DeviceMemory` instead of `Size`, since "size" is a bit ambiguous.